kube/src/test/java/org/openshift/ping/kube/test/SimplePingTest.java

```diff
@@ -18,13 +18,15 @@
 
 import org.jgroups.protocols.FILE_PING;
 import org.jgroups.stack.Protocol;
+import org.junit.Ignore;
 
 /**
  * A parallel test to ZK tests.
  * Just so we know tests are fine.
  *
  * @author <a href="mailto:ales.justin@jboss.org">Ales Justin</a>
  */
+@Ignore("To be clarified with Bela")
```

Comment:
Have you read [1]? JGroups does not necessarily update the file on every membership change...
[1] http://www.jgroups.org/manual4/index.html#FILE_PING